maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -222,7 +229,7 @@ private static void expandJar(Jar jar, File dir) throws Exception {
 					continue;
 			}
 
-			IO.copy(resource.openInputStream(), outFile);
+			IO.copy(resource.openInputStream(), buildContext.newFileOutputStream(outFile));
```

Comment:
The output stream is not closed here. The prior code created and closed the output stream. This code creates an output stream but never closes it.